zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -6011,6 +6042,37 @@ components:
         processDefinitionName:
           type: string
           description: The name of the process definition (fall backs to the process definition ID if not available).
+    ProcessInstanceSequenceFlowsQueryResult:
+      description: Process instance sequence flows query response.
+      type: object
+      properties:
+        items:
+          description: The sequence flows.
+          type: array
+          items:
+            $ref: "#/components/schemas/ProcessInstanceSequenceFlowResult"
+    ProcessInstanceSequenceFlowResult:
+      description: Process instance sequence flows result.
+      type: object
+      properties:
+        sequenceFlowId:
+          type: string
+          description: The sequence flow ID.
+        processInstanceKey:
+          type: string
+          description: The key of this process instance.
+        processDefinitionKey:
+          type: string
+          description: The process definition key.
+        processDefinitionId:
+          type: string
+          description: The process definition ID.
+        elementId:
+          type: string
+          description: The element ID for this element instance.
```

❓ This is the sequence flow's ID from the model. So, "elementId" is correct, but the description could be more helpful.

```suggestion
        elementId:
          type: string
          description: The element ID for this sequence flow, as provided in the BPMN process.
```